config/functions

```diff
@@ -217,6 +209,33 @@ strip_gold() {
   LDFLAGS=`echo $LDFLAGS | sed -e "s|-fuse-ld=gold||g"`
 }
 
+get_projectdir() {
+  local d_list=( # main directory roots to search
+    $PROJECT_DIR/$PROJECT/packages
+    $HOME/.openelec/projects/$PROJECT/packages
+    $HOME/.openelec/packages
+    $PROJECT_DIR/$PROJECT
+    $HOME/.openelec/projects/$PROJECT
+    $HOME/.openelec
```

Comment:
I could potentially add $PKG_DIR to the main d_list variable thereby enabling searching inside the original PKG_DIR and possible removing the need to do an else.  Not sure if this makes it more confusing or not.